kafka/channel/pkg/dispatcher/dispatcher.go

```diff
@@ -353,7 +353,7 @@ func fromKafkaMessage(ctx context.Context, kafkaMessage *sarama.ConsumerMessage)
 			event.SetDataSchema(v)
 		default:
 			// Extensions
-			if IsAlphaNumericLowercaseLetters(h) {
+			if IsAlphaNumeric(h) {
```

Comment:
This is a behavior change: previously a header with uppercase letters would not be set as an extension. Now it will (and I believe sdk-go will then lowercase it). @n3wscott @matzew any thoughts on this change?

Reply:
The repo doesn't build without this change. It's due to this breaking change: https://github.com/cloudevents/sdk-go/pull/274